piqe_ocp_lib/api/tasks/operator_ops.py

```diff
@@ -77,15 +78,24 @@ def add_operator_to_cluster(self, operator_name, target_namespaces=[]):
 
         return True
 
+    def is_operator_installed(self, operator_name, operator_namespace):
```

Comment:
```suggestion
    def is_operator_installed(self, operator_name: str, operator_namespace: str) -> bool:
```

Reply:
Not a hard requirement at this moment.

Reply:
@adtandale I understand, I thought it would be best to follow the conventions already present in the script but yes, not a hard requirement. https://github.com/piqe-test-libraries/piqe-ocp-lib/pull/50/files/5424966e6dc10af4a1e69348e9af06a2acb98045#diff-8e09d2694e83a79f8723db781697c1babc7e0663cfd7ec14de5c7a9b72c01394R92